### PR TITLE
Nitpicky cleanups

### PR DESCRIPTION
**Pull request by AndyZe**
Some nitpicky cleanups that I noticed while checking out the codebase  :+1:

### CODE REVIEW

File: src/curobo/curobolib/cpp/geom_cuda.cpp [hunk 1]
```diff
@@ -32,8 +32,6 @@ std::vector<torch::Tensor>self_collision_distance(
   const int           ndpt         = 8, // Does this need to match template?
   const bool          debug        = false);
 
-// CUDA forward declarations
```
Comment (AndyZe) [inline src/curobo/curobolib/cpp/geom_cuda.cpp:35]: Repeated at L17

File: src/curobo/curobolib/cpp/self_collision_kernel.cu [hunk 2]
```diff
@@ -175,8 +175,6 @@ namespace Curobo
       {
         sph2 = __rs_shared[j]; // we need not load sph2 in every iteration.
 
-#pragma unroll 16
```
Comment (AndyZe) [inline src/curobo/curobolib/cpp/self_collision_kernel.cu:178]: I don't believe ndpt is known at compile time?

**Comment by AndyZe** [comment]
Pinging for review. Should be quick, I think

File: src/curobo/curobolib/cpp/self_collision_kernel.cu (hunk 2 above)
Comment (balakumar-s) [reply src/curobo/curobolib/cpp/self_collision_kernel.cu:178]: ndpt is not known. You are correct.

**Comment by balakumar-s** [comment]
Thanks! I will merge this today.

**Pull request merged**